cdap-proto/src/main/java/co/cask/cdap/proto/artifact/AppRequest.java

```diff
@@ -41,19 +41,19 @@ public AppRequest(ArtifactSummary artifact) {
   }
 
   public AppRequest(ArtifactSummary artifact, @Nullable T config) {
-    this(artifact, config, null, null, true);
+    this(artifact, config, null, null, null);
```

Comment:
But this has a default behavior if not specified the value should represent that in my opinion. Also it's a Boolean so expectation is that it will true/false. Null can cause issues.